debezium-server/README.md

```diff
@@ -19,3 +19,34 @@ It is thus recommended to execute integration tests per-module and set-up necess
 * Create a new topic `gcloud pubsub topics create testc.inventory.customers`
 * Build the module and execute the tests `mvn clean install -DskipITs=false -am -pl debezium-server-pubsub`
 * Remove the topic `gcloud pubsub topics delete testc.inventory.customers`
+
+## Azure Event Hubs
+
+### Create an Event Hubs namespace
+
+Create an [Event Hubs namespace](https://docs.microsoft.com/azure/event-hubs/event-hubs-features?WT.mc_id=debezium-docs-abhishgu#namespace). Check the documentation for options on how do this using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create?WT.mc_id=debezium-docs-abhishgu#create-an-event-hubs-namespace), [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli?WT.mc_id=debezium-docs-abhishgu#create-an-event-hubs-namespace) etc.
+
+### Create an Event Hub
+
+Create an Event Hub (equivalent to a topic) with `one` partition. Check the documentation for options on how do this using the [Azure Portal](https://docs.microsoft.com/azure/event-hubs/event-hubs-create?WT.mc_id=debezium-docs-abhishgu#create-an-event-hub), [Azure CLI](https://docs.microsoft.com/azure/event-hubs/event-hubs-quickstart-cli?WT.mc_id=debezium-docs-abhishgu#create-an-event-hub) etc. 
+
+### Build the module
+
+[Get the Connection string](https://docs.microsoft.com/azure/event-hubs/event-hubs-get-connection-string?WT.mc_id=debezium-docs-abhishgu) required to communicate with Event Hubs. The format is: `Endpoint=sb://<NAMESPACE>/;SharedAccessKeyName=<ACCESS_KEY_NAME>;SharedAccessKey=<ACCESS_KEY_VALUE>`
+
+Set environment variables required for tests:
+
+```shell
+export EVENTHUBS_CONNECTION_STRING=<Event Hubs connection string>
+export EVENTHUBS_NAME=<name of the Event hub created in previous step>
+```
+
+Execute the tests:
+
+```shell
+mvn clean install -DskipITs=false -Deventhubs.connection.string=$EVENTHUBS_CONNECTION_STRING -Deventhubs.hub.name=$EVENTHUBS_NAME -am -pl debezium-server-eventhubs
```

Comment:
Hey @abhirockzz, just testing this. It sems to work, the test passes. Is there any way for me to examine the contents of the topic (or "Event Hub")?

Reply:
@gunnarmorling you're looking for a way to inspect the actual topic contents outside of the test, right? Like a standalone app?

Reply:
Yes, or just even the right az CLI command so I can take a look at the emitted events.

Reply:
If your Event Hubs namespace in Kafka enabled (in case of Standard tier, it's automatically enabled), may I suggest using `kafkacat`?

Create `kafkacat.conf`

```
metadata.broker.list=<namespace>.servicebus.windows.net:9093
security.protocol=SASL_SSL
sasl.mechanisms=PLAIN
sasl.username=$ConnectionString
sasl.password=Endpoint=sb://<namespace>.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=<access key>
```

Start consuming events from topic/hub

```
export KAFKACAT_CONFIG=<path to kafkacat.conf>
kafkacat -b <namespace>.servicebus.windows.net:9093 -t my-test-hub
```

Run the IT

Reply:
Excellent, thanks a lot. Can confirm I see the events 👍